### PR TITLE
feat: update account methods

### PR DESCRIPTION
**Pull request by Tuditi**
# Description of change

Added missing method (getIncomingTransactionData) and updated a type on the nodejs bindings.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/nodejs/lib/Account.ts [hunk 1]
```diff
@@ -515,7 +533,7 @@ export class Account {
      * @returns The incoming transactions with their inputs.
      */
     async incomingTransactions(): Promise<
-        [string, [ITransactionPayload, IOutputResponse[]]][]
+        [string, IncomingTransactionData][]
```
Comment (Thoralf-M) [inline bindings/nodejs/lib/Account.ts:536]: I think the IOutputResponse needs to stay, it's the inputs of the transaction if still available

Comment (Tuditi) [reply bindings/nodejs/lib/Account.ts:536]: I just tried to follow what you returned inside the rs code (https://github.com/iotaledger/wallet.rs/blob/feat/update-account-methods/src/message_interface/response.rs#L29). I think Thibault changed it to return `IOutputMetadataResponse` and I created my own type to make it a bit more readable.

Comment (Thoralf-M) [reply bindings/nodejs/lib/Account.ts:536]: Ah sorry missed that it's a new type

**Review by Thoralf-M — CHANGES_REQUESTED** [review]
Missing change file

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**